crates/uv/src/main.rs

```diff
@@ -105,6 +105,9 @@ async fn run() -> Result<ExitStatus> {
         }
     };
 
+    // Load the workspace settings.
+    let _ = uv_workspace::Workspace::find(env::current_dir()?)?;
```

Comment:
If we don't have read permissions for the current directory should we error or just not attempt to read a workspace there?